docs/faq/design-systems.md

```diff
@@ -11,3 +12,6 @@ The hardest part of any project is often overcoming that first blank file and ge
 - [Morningstar](http://designsystem.morningstar.com/components/component-status.html)
 - [SAP UI5](https://sap.github.io/ui5-webcomponents/)
 - [Vaadin](https://vaadin.com/components)
+
+## Don't see your design system here?
```

Comment:
Should we do it the same as [here](https://github.com/open-wc/open-wc/pull/351/files) for consistency? Or do you think we should change the way I did it there, to the way you did it here? Thoughts?

```
::: tip Looking for contributors!
We're actively looking to extend this page with common best practices. If you have anything to contribute, please make an issue or a pull request on our [github repository](https://github.com/open-wc/).
:::
```